package/webinterface-onboot/package

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+_pkgname='webinterface-onboot'
+pkgnames=("$_pkgname")
+pkgdesc="Start the web interface without the cable"
+url="https://github.com/rM-self-serve/$_pkgname"
+pkgver=1.2.2-1
+timestamp=2023-12-03T11:43:00Z
+section="utils"
+maintainer="rM-self-serve <122753594+rM-self-serve@users.noreply.github.com>"
+license=MIT
+
+_pkgalias="webint-onboot"
+
+source=(
+    "$url"/archive/cbb2b936f0d27898701695f4d166dcf01b0174e2.zip
+    "$_pkgname-toltec.service"
+)
+sha256sums=(
+    053de1cb3c889df844f6c3e9ce1947803663a13b5efd3bd376a1937c0d1c3787
+    SKIP
+)
+
+package() {
+    install -D -m 755 -t "$pkgdir/opt/bin" "$srcdir/$_pkgname"
+    ln -s "/opt/bin/$_pkgname" "$pkgdir/opt/bin/$_pkgalias"
+}
+
+_restore() {
+    if webinterface-onboot is-hack-applied > /dev/null 2>&1; then
+        echo "Reverting /usr/bin/xochitl"
+        if ! webinterface-onboot revert-hack --backup -y > /dev/null 2>&1; then
+            if ! webinterface-onboot revert-hack --reverse -y > /dev/null 2>&1; then
+                echo "Could not revert /usr/bin/xochitl"
+            fi
+        fi
+        echo "Success"
+    fi
+    if webinterface-onboot is-prstip-applied > /dev/null 2>&1; then
+        echo "Reverting /etc/ifplugd/ifplugd.action"
+        if ! webinterface-onboot revert-prstip -y > /dev/null 2>&1; then
+            echo "Could not revert /etc/ifplugd/ifplugd.action"
```

Comment:
Is this safe to ignore? I would assume we would want to error out and have the user investigate what went wrong, so that opkg keeps track of the partially broken state and will reattempt the next install.